examples/basic/annotations/whisker.py

```diff
@@ -1,3 +1,14 @@
+''' A marker plot that shows the relationship between Car class and HWY from the autommpg sample data. 
+This example demonstrates the use of Whisker to display quantile ranges in the plot.
```

Comment:
I think "class" here might be misunderstood as an OO "class"

```suggestion
''' A marker plot that shows the relationship between car type and highway MPG from the autompg
sample data.  This example demonstrates the use of whiskers to display quantile ranges in the plot.
```